Fastfile

```diff
@@ -164,14 +157,24 @@ def _get_team_id()
   team_id
 end
 
-def _parse_provision_file()
-  sh 'RUBYSCRIPT="$(echo \'def target(targetName, &doBlock)
-    profileName = doBlock.call
-    print targetName + "_" + "PROVISIONING_PROFILE = " + profileName + "\n"
+def _create_xcconfig(filename)
+  # TODO: Replace spaces with underscores?
+  # TODO: Try not to need `TAB_PROJECT_PATH`
+  project = Xcodeproj::Project.open(ENV['TAB_PROJECT_PATH'])
```

Comment:
Could you still use `FL_PROJECT_SIGNING_PROJECT_PATH` by doing
```ruby
project = Xcodeproj::Project.open("../#{ENV['FL_PROJECT_SIGNING_PROJECT_PATH']}")
```
?

Reply:
I could, but what if people have their Fastlane directory somewhere else?

Reply:
V good point